glimpse/src/main/java/edu/asu/diging/glimpse/service/FileInterface.java

```diff
@@ -0,0 +1,8 @@
+package edu.asu.diging.glimpse.service;
+
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileInterface {

```

Comment:
interface and class shoulnd't have the same name. Typically you either call your interface ```IMyService``` and your implemenation ```MyService```, or your interface ```MyService``` and its implementation ```MyServiceImpl```.